src/cmake_modules/FindROCm.cmake

```diff
@@ -16,25 +16,12 @@
 
 find_path(
   ROCM_INCLUDE_DIRS
-    hsa.h
+    hsa/hsa.h
   HINTS
     ${ROC_DIR}/include
-    ${ROC_DIR}/include/hsa
-    ${ROC_DIR}/hsa/include
-    ${ROC_DIR}/hsa/include/hsa
-    ${ROC_DIR}/hsa
-    ${ROC_DIR}
     ${ROCR_DIR}/include
-    ${ROCR_DIR}/include/hsa
-    ${ROCR_DIR}/hsa/include
-    ${ROCR_DIR}/hsa/include/hsa
-    ${ROCR_DIR}
     /opt/rocm/include
-    /opt/rocm/hsa/include
-    /usr/local/include
```

Comment:
Can we let /use/local/include remain as a hint path? Any reason why we do not want this as a hint path?

Reply:
Its for making sure that the "hsa.h" file search should be in rocm folder.  We are expecting the ROCM include path for hsa.h